### PR TITLE
[Cogs Face] Update Face SDK‘s auto-generated code to 3.0.0-preview with latest swagger.

### PR DESCRIPTION
**Pull request by lebronJ**
Regenerate Face SDK with latest spec Azure/azure-rest-api-specs#3552
* Add & Update unit tests (regenerate unit-test session records).
* Increment package version to 3.0.0-preview.
* Add release notes.
* This has been internally reviewed (lebronJ/azure-sdk-for-net#2) inside Face SDK team.

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by huxuan** [comment]
LGTM.

**Review by yangyuan — APPROVED** [review]
LGTM

File: src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face/Microsoft.Azure.CognitiveServices.Vision.Face.csproj [hunk 1]
```diff
@@ -6,14 +6,15 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Vision.Face</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services Face APIs.</Description>
-    <Version>2.1.0-preview</Version>
+    <Version>3.0.0-preview</Version>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face/Microsoft.Azure.CognitiveServices.Vision.Face.csproj]: Is there a reason why this is a major version bump?

Comment (dsgouda) [reply src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face/Microsoft.Azure.CognitiveServices.Vision.Face.csproj]: Please create a properties.cs similar to [this](https://github.com/lebronJ/azure-sdk-for-net/blob/0e3f092cf127d84eeacc49325604cd7fa5703bef/src/SDKs/CognitiveServices/dataPlane/Search/BingCustomSearch/BingCustomSearch/Properties/AssemblyInfo.cs) and assign the appropriate version numbers

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face/Microsoft.Azure.CognitiveServices.Vision.Face.csproj (hunk 1 above)
Comment (huxuan) [reply src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face/Microsoft.Azure.CognitiveServices.Vision.Face.csproj]: By adding support of LargeFaceList and LargePersonGroup, there are some breaking changes in those operations like FindSimilar, Identify and Verify. So we propose a major version upgrade.

Comment (dsgouda) [reply src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face/Microsoft.Azure.CognitiveServices.Vision.Face.csproj]: When in preview, you can release a breaking change with a minor version bump, i.e., `2.2.0-preview`

Comment (huxuan) [reply src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face/Microsoft.Azure.CognitiveServices.Vision.Face.csproj]: OK, we will follow that rule. Thanks for the information.

Comment (lebronJ) [reply src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face/Microsoft.Azure.CognitiveServices.Vision.Face.csproj]: Resolved by adding `properties.cs` and change version to `2.2.0-preview`. Regenerate SDK by master swagger and regenerate test records. Thanks.

**Review by dsgouda — APPROVED** [review]

**Comment by dsgouda** [comment]
LGTM

**Pull request merged**